objectivec/GPBBootstrap.h

```diff
@@ -28,44 +28,61 @@
 // (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
 // OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-// The Objective C runtime has complete enough info that most protos don’t end
-// up using this, so leaving it on is no cost or very little cost.  If you
-// happen to see it causing bloat, this is the way to disable it. If you do
-// need to disable it, try only disabling it for Release builds as having
-// full TextFormat can be useful for debugging.
+/**
+ * The Objective C runtime has complete enough info that most protos don’t end
+ * up using this, so leaving it on is no cost or very little cost.  If you
+ * happen to see it causing bloat, this is the way to disable it. If you do
+ * need to disable it, try only disabling it for Release builds as having
+ * full TextFormat can be useful for debugging.
+ **/
 #ifndef GPBOBJC_SKIP_MESSAGE_TEXTFORMAT_EXTRAS
 #define GPBOBJC_SKIP_MESSAGE_TEXTFORMAT_EXTRAS 0
 #endif
 
-// Used in the generated code to give sizes to enums. int32_t was chosen based
-// on the fact that Protocol Buffers enums are limited to this range.
+/**
+ * Used in the generated code to give sizes to enums. int32_t was chosen based
+ * on the fact that Protocol Buffers enums are limited to this range.
+ **/
 #if !__has_feature(objc_fixed_enum)
  #error All supported Xcode versions should support objc_fixed_enum.
 #endif
-// If the headers are imported into Objective-C++, we can run into an issue
-// where the defintion of NS_ENUM (really CF_ENUM) changes based on the C++
-// standard that is in effect.  If it isn't C++11 or higher, the definition
-// doesn't allow us to forward declare. We work around this one case by
-// providing a local definition. The default case has to use NS_ENUM for the
-// magic that is Swift bridging of enums.
+
+/**
+ * If the headers are imported into Objective-C++, we can run into an issue
+ * where the defintion of NS_ENUM (really CF_ENUM) changes based on the C++
+ * standard that is in effect.  If it isn't C++11 or higher, the definition
+ * doesn't allow us to forward declare. We work around this one case by
+ * providing a local definition. The default case has to use NS_ENUM for the
+ * magic that is Swift bridging of enums.
+ **/
```

Comment:
Where does this show up?  Should it stay // and do we need a real comment for what GPB_ENUM is for cocoadocs?